src/webui/www/private/views/rss.html

```diff
@@ -406,10 +405,11 @@
             if (path === "")
                 visibleArticles = visibleArticles.filter((a) => !a.isRead);
 
+            let rowCount = -1;
```

Comment:
Now these variables don't actually mean "row count". Or do you believe I renamed them in my code suggestion for nothing?

Reply:
The variable renaming is out of the scope for this PR. I would prefer not to get involved with it.

Reply:
sigh. Done.